db/migrate/20150714140850_remove_new_from_compute_attributes.rb

```diff
@@ -0,0 +1,35 @@
+class FakeComputeAttribute < ActiveRecord::Base
+  set_table_name 'compute_attributes'
+  serialize :vm_attrs, Hash
+end
+
+class RemoveNewFromComputeAttributes < ActiveRecord::Migration
+  def up
+    FakeComputeAttribute.all.each do |comp_attr|
+      attrs_name = ['nics', 'interfaces'].find do |name|
+        comp_attr.vm_attrs["#{name}_attributes"].present?
+      end
+      comp_attr.vm_attrs["#{attrs_name}_attributes"].delete("new_#{attrs_name}") if attrs_name
+      comp_attr.vm_attrs['volumes_attributes'].delete('new_volumes')
+      comp_attr.save!
+    end
+  end
+
+  def down
```

Comment:
what's the purpose of migration this way? you can't restore the original data from new_* so you're copying attributes of first interface? I'd probably keep the down direction empty